src/components/catalog-parameters/catalog-parameters.controller.ts

```diff
@@ -47,6 +49,30 @@ export class CatalogParametersController implements angular.IController {
     }
   }
 
+  private setupFileSchema() {
+    // Find any parameters in the form definition that have a type of 'file'
+    _.each(this.ctrl.parameterFormDefinition, (parameter: any) => {
+      _.each(_.get(parameter, 'items'), (item: any, index: any) => {
+        if (item.type && item.type === 'file') {
+          // Need to set this to 'string' so the input type is also string (not file)
+          // angular-schema-form-base64-file-upload add-on expects this.
+          parameter.items[index].type = 'string';
+
+          // angular-schema-form-base64-file-upload add-on expects:
+          // * the type to be 'string'
+          // * and a format field set to 'base64'
+          // * a maxSize
+          // https://github.com/Textalk/angular-schema-form-base64-file-upload#usage
+          _.assign(_.get(this.ctrl.parameterSchema, ['properties', item.key]), {
+            "type": "string",
+            "format": "base64",
+            "maxSize": '5242880' // 5MB
```

Comment:
@pb82 @weileylee What are the likely sizes for p12 ios certs, in terms of putting a max limit on the upload?